lib/Factory/MediaFactory.php

```diff
@@ -634,7 +634,7 @@ public function query($sortOrder = null, $filterBy = [])
         }
 
         if ($this->getSanitizer()->getInt('imageProcessing', $filterBy) !== null) {
-            $body .= 'AND ( media.type = \'image\' OR media.type = \'module\' ) ';
+            $body .= 'AND ( media.type = \'image\' OR media.type = \'module\' AND media.moduleSystemFile = 0 ) ';
```

Comment:
sorry but this still isn't right - you need extra `()` around the 2nd and 3rd criteria

```
$body .= 'AND ( media.type = \'image\' OR (media.type = \'module\' AND media.moduleSystemFile = 0) ) ';
```